docs/scipp-neutron/from-mantid-to-scipp.ipynb

```diff
@@ -594,6 +618,21 @@
     "Bin edges in scipp can be created from an arbitrary array with increasing values, the use of `numpy.geomspace` is simply one example for generating bins spaced evenly on a log scale."
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<div class=\"alert alert-info\">\n",
+    "\n",
+    "**Note**\n",
+    "\n",
+    "That both scipp and Mantid support binary and inplace operations such as + and +=.\n",
+    "Mantid's binary operations call underlying algorithms as part of implementation.\n",
+    "This makes it difficult to change some default behaviour, for example if you want to prevent output workspaces from being registered in Mantids Analysis Data Service.\n",
```

Comment:
```suggestion
    "This makes it difficult to change some default behaviour, for example if you want to prevent output workspaces from being registered in Mantid's Analysis Data Service.\n",
```